src/server/components/sitemap/controllers.ts

```diff
@@ -23,5 +25,6 @@ export function sitemapController(_req: Request, res: Response): void {
 
   res.render("sitemap", {
     sections,
+    pageTitle: pageTitles[sitemapRoute],
```

Comment:
Optional change but line **28** can be re-written like this:
```
pageTitle: pageTitles.sitemapRoute,
```

Reply:
changes made